### ISSUE TITLE
Switching between routes in Angular 8 doesnt stop the video playing

### ISSUE THREAD

**Issue by rbenzing** [jepiqueau/capacitor-video-player#12]
I am embedding for PWA and noticing that the video doesnt stop when going back in the routes using routerLink how do i destroy the component on switching views?

Ionic 4
Angular 8
Capacitor only

**Comment by jepiqueau** [comment]
@rbenzing First thanks to use the plugin. this has been fixed in the release 2.0.1-4. A new method `stopAllPlayers()` has been added, you can call it before to leave the route like this
```
    async ngOnDestroy() {
        const res:any  = await this._videoPlayer.stopAllPlayers();
    }
```
If you have some timeout function running do not forget to clear them before calling the stopAllPlayers() method.

I let you close the issue after you implemented successfully in your PWA app.

**Comment by jepiqueau** [comment]
@rbenzing Any chance that you have tested it
